### ISSUE TITLE
Uptime monitoring for all .gov domains

### ISSUE THREAD

**Issue by daguar** [codeforamerica/project-ideas#51]
We've got a prototype for monitoring all state-level SNAP web sites (SNAP Status Board, #43 , http://uptime.statuscake.com/?TestID=pQYuAW4tAi )

and 18F just released all the .gov domains out there: https://18f.gsa.gov/2014/12/18/a-complete-list-of-gov-domains/ 

Once we've figured out all the kinks of external monitoring with SNAP sites, let's extend it to _all_ .gov sites.

cc @konklone @gbinal

**Comment by konklone** [comment]
Something to bear in mind - many of the domains on the `.gov` list aren't used for websites, or may be (currently) unused in their entirety.

**Comment by daguar** [comment]
Thanks for the note @konklone; good to know.

**Comment by migurski** [comment]
Yeah, this is kind of a shotgun approach and might not be appropriate.

**Comment by asweigart** [comment]
Something that could help is this csv file of all .gov domain names: https://18f.gsa.gov/2014/12/18/a-complete-list-of-gov-domains/

**Comment by konklone** [comment]
Here's a perhaps relevant project my team at 18F released this week: https://pulse.cio.gov
